src/testers.jl

```diff
@@ -197,7 +211,8 @@ end
 - `ẋ`: differential w.r.t. `x` (should generally be set randomly).
 
 `fkwargs` are passed to `f` as keyword arguments.
-If `check_inferred=true`, then the type-stability of the `frule` is checked.
+If `check_inferred=true`, then the type-stability of the `frule` is checked, as long as `f`
+is itself type-stable.
```

Comment:
similar here (I should have caught this before).
Inferable implies type stable
type stable does not imply inferable.